core/src/test/java/org/elasticsearch/cluster/metadata/IndexNameExpressionResolverTests.java

```diff
@@ -643,6 +643,33 @@ public void testConcreteIndicesWildcardWithNegation() {
         assertEquals(0, indexNames.length);
     }
 
+    // issue #23960
```

Comment:
In general, I would prefer a straightforward comment explaining the issue being tested. A link to a GitHub issues means that I have to be online and context switch to obtain more information.